common/src/IO/ObjSerializer.cpp

```diff
@@ -78,17 +80,18 @@ namespace TrenchBroom {
             return str;
         }
 
-        ObjSerializer::ObjSerializer(std::ostream& objStream, std::ostream& mtlStream, const Path& mtlPath) :
+        ObjSerializer::ObjSerializer(std::ostream& objStream, std::ostream& mtlStream, const Path& mtlPath, std::shared_ptr<IO::ObjExportOptions>  options) :
```

Comment:
If you use `std::variant` for the export options, you don't need to heap allocate and can do away with the pointers. Just take `ObjExportOptions` by value and move it in place.